modules/logging-http-forward.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// *
+
+:_content-type: PROCEDURE
+[id="logging-deploy-loki-console_{context}"]
+_content-type: PROCEDURE
+= Forwarding logs over HTTP
+
+Forwarding logs over HTTP is supported for both fluentd and vector collectors. To enable, specify `http` as the output type in the `ClusterLogForwarder` custom resource (CR).
+
+* Create or edit the ClusterLogForwarder Custom Resource (CR) using the template below:
+
+.Example ClusterLogForwarder CR
+[source,yaml]
+----
+apiVersion: "logging.openshift.io/v1"
+kind: "ClusterLogForwarder"
+metadata:
+  name: "instance"
+  namespace: "openshift-logging"
+spec:
+  outputs:
+    - name: httpout-app
+      type: http
+      url: <1>
+      http:
+        headers: <2>
+          h1: v1
+          h2: v2
+        method: POST
+        secret:
+          name: <3>
```

Comment:
@libander just curious if there is a tls has no value in this example? I fit's important to call out it's the destination TLS setting, then I take it customers would use that setting. Otherwise, this LGTM.